themes/default/content/docs/intro/pulumi-service/ci-cd-integration-assistant.md

```diff
@@ -17,46 +17,31 @@ to [Organizations]({{< relref "/docs/intro/pulumi-service/organizations" >}}), n
 
 <!--more-->
 
-A version control system (VCS) provides team collaboration capabilities and ensures that the source code for your Pulumi project is not
-just on a single developer's machine. Using a CI/CD system makes your team more productive, by automatically deploying your Pulumi stacks.
-So that code is delivered to production -- meaning less of a lag between commits and what your end users are seeing.
-
-The assistant contains a guided experience to help teams configure a VCS as well as automate it with a
+A version control system (VCS) provides team collaboration capabilities and ensures that the source code for your Pulumi project is not on a single developer's machine. Using a CI/CD system makes your team more productive, by automatically deploying your Pulumi stacks. The assistant contains a guided experience to help teams configure a VCS as well as automate it with a
 CI/CD pipeline.
 
 The assistant has two options to help you to get started with integrations:
 
 * Use a single service that provides both a version control system and CI/CD pipelines
-* Or, just get the starter workflow for a CI/CD service of your choice customized for your stack
+* Or, get the starter workflow for a CI/CD service of your choice customized for your stack
```

Comment:
Why would I pick the first vs the second? Not sure if those docs have already explained this. =)

Reply:
🤷🏻 can try and fix

Reply:
okay what that meant was that you can use the same service for both version control and CI/CD or you can use different services for those. and the assistant supports both. ive tried to make this more clear.

Reply:
why you would do that i think depends on your org needs and things